### DISCUSSION TITLE
Errore cURL #28: Resolving timed out after 5000 milliseconds

### DISCUSSION THREAD

**Discussion by donlegro** [fattureincloud/api#468] [Q&A] [answered]
Salve, da circa una settimana quando i miei clienti provano a fare chiamate API dal mio gestionale, ricevono il messaggio in oggetto. Sia invio che ricezione di acquisti o emessi.
Ho provato anche ad allungare fino a 30000 millisecondi ma il risultato non cambia.
Il fatto che non succeda sempre ma molto spesso, mi viene da pensare che forse è stato abbassato un limite che prima non superavo e ora sì oppure è aumentato il flusso trasmesso e quindi è da cambiare qualche parametro.
Che controlli posso fare? 
Grazie in anticipo.

P.S. Essendo il fornitore del gestionale non ho accesso al portale Fatture in cloud dei vari clienti, quindi non posso verificare eventuali parametri.

**Comment by emajo** [comment, marked as answer]
Ciao @donlegro mi sembra molto strano dei timeout così alti, anche perché lato nostro abbiamo un timeout a 30 secondi di esecuzione, quindi impossibile che dia timeout al client con 30000 millisecondi, consiglio di scrivere al nostro supporto clienti specificando il company id dell'azienda con cui sta ricevendo l'errore così da fare controlli più approfonditi.

Grazie,
Emanuele.

**Reply by valmoz** [reply]
Buongiorno @donlegro è riuscito a risolvere?
Grazie

Mauro

**Comment by donlegro** [comment]
Grazie Mauro, stavo temporeggiando per vedere se si ripresentava.
Probabilmente (non ho certezza purtroppo) alcuni clienti avevano superato il limite mensile e forse gestivo male il messaggio di ritorno. 
Invece di bloccare continuavo a inviare chiamate API e quindi andava in timeout. Ora ho inserito dei controlli più precisi.

**Reply by valmoz** [reply]
Ok grazie.
Per il momento chiudo questa discussione, qualora dovesse essere necessario potrà riaprirla in seguito.

Grazie
Mauro
